cloud-haven-react-app/src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import './SignUp.css';
+
+const SignUp = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    
+    const handleSubmit = (evt) => {
+        evt.preventDefault();
+        var signUpData = {
+            "email": email,
+            "password": password
+        };
+        alert(JSON.stringify(signUpData))
+    }
+
+    return (
+        <div className="signupWrapper center" >
+        <h1 className="center">Create an account</h1>
+        <form>
+            <div className="inputBox center centerText">
+                <label>Email Address</label>
+                <br></br>
+                <input 
+                    type="email" 
+                    name="email" required
+                    onChange={e => setEmail(e.target.value)}
+                />
+            </div>
+            <div className="inputBox center centerText">
+                <label>Password</label>
+                <br></br>
+                <input 
+                    type="password" 
+                    name="password1"
```

Comment:
```suggestion
                    name="password"
```
I think this might be a typo. Is there a reason it was called "password1?"